### DISCUSSION TITLE
Shared Play Count Feature

### DISCUSSION THREAD

**Discussion by thebiblelover7** [navidrome/navidrome#3131] [Ideas]
Would it be possible to have a configuration option to share play count across multiple users? 
It would be pretty cool in our use case to have a shared play count so that "popular" songs will work a bit more like Spotify. That way, recommendations done by Feishin or other clients may be more useful.
I understand that some people may not want to have this enabled.

<img width="871" alt="Screenshot 2024-07-02 at 4 23 14 PM" src="https://github.com/navidrome/navidrome/assets/61815862/56194076-b6e7-4704-898b-0965622f8f96">

**Comment by deluan** [comment]
I think this would be better to be implemented as a new field in the [OpenSubsonic API](https://github.com/opensubsonic/open-subsonic-api/discussions/categories/general), as this could break the expectations of most clients out there. I suggest you to open a discussion in the forum above.

**Reply by thebiblelover7** [reply]
Sounds good! I've created one there. opensubsonic/open-subsonic-api#96

**Comment by noah1234j** [comment]
Ive upvoted this there as well. Maybe we could have a config switch that we flip where the play count to global or by user
